src/NoteMouseHandler/SelectionMouseHandler.js

```diff
@@ -75,12 +74,12 @@ const moveSelectionAction = ctx => (onMouseDown, onMouseMove) => {
   const originalSelection = _.clone(ctx.selection)
   let startPosition
 
-  onMouseDown(e => {
-    startPosition = getLocal(e)
+  onMouseDown(local => {
+    startPosition = local
   })
 
-  onMouseMove(e => {
-    const movement = pointSub(getLocal(e), startPosition)
+  onMouseMove(local => {
+    const movement = pointSub(local, startPosition)
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->